gnssanalysis/gn_download.py

```diff
@@ -252,7 +276,7 @@ def check_file_present(comp_filename: str, dwndir: str) -> bool:
     return present
 
 
-def check_n_download_url(url, dwndir, filename=False):
+def check_n_download_url(url, dwndir, filename=False, uncomp=False):
```

Comment:
Yeah, I don't mind changing the name, I was thinking of changing all of them, but that would also involve a change to `ginan-ops` code that uses some of these functions. 
Also, `uncompress` just because that's what the utility invoked is called, but I can change to decompress